### PR TITLE
javascript/processor: better error messages for timeout/max-size

### PR DESCRIPTION
**Pull request by mattrobenolt**
@getsentry/platform

### CODE REVIEW

File: src/sentry/models/eventerror.py [hunk 1]
```diff
@@ -42,8 +42,8 @@ class EventError(object):
         JS_TOO_MANY_REMOTE_SOURCES: u'The maximum number of remote source requests was made',
         JS_INVALID_SOURCE_ENCODING: u'Source file was not \'{value}\' encoding: {url}',
         JS_INVALID_SOURCEMAP_LOCATION: u'Invalid location in sourcemap: ({column}, {row})',
-        JS_TOO_LARGE: u'Remote file too large',
-        JS_FETCH_TIMEOUT: u'Remote file took too long to load',
+        JS_TOO_LARGE: u'Remote file too large: ({max_size}, {url})',
+        JS_FETCH_TIMEOUT: u'Remote file took too long to load: ({timeout}, {url})',
```
Comment (JTCunning) [inline src/sentry/models/eventerror.py]: I'd add the time units in.

File: src/sentry/models/eventerror.py [hunk 2]
```diff
@@ -42,8 +42,8 @@ class EventError(object):
         JS_TOO_MANY_REMOTE_SOURCES: u'The maximum number of remote source requests was made',
         JS_INVALID_SOURCE_ENCODING: u'Source file was not \'{value}\' encoding: {url}',
         JS_INVALID_SOURCEMAP_LOCATION: u'Invalid location in sourcemap: ({column}, {row})',
-        JS_TOO_LARGE: u'Remote file too large',
-        JS_FETCH_TIMEOUT: u'Remote file took too long to load',
+        JS_TOO_LARGE: u'Remote file too large: ({max_size}, {url})',
```
Comment (mattrobenolt) [inline src/sentry/models/eventerror.py]: Wait, can you do that?

Comment (JTCunning) [inline src/sentry/models/eventerror.py]: Illegal operations god.

File: src/sentry/lang/javascript/processor.py [hunk 3]
```diff
@@ -409,11 +409,13 @@ def fetch_file(url, project=None, release=None, allow_scraping=True):
                         error = {
                             'type': EventError.JS_FETCH_TIMEOUT,
                             'url': expose_url(url),
+                            'timeout': settings.SENTRY_SOURCE_FETCH_TIMEOUT,
                         }
                     elif isinstance(exc, OverflowError):
                         error = {
                             'type': EventError.JS_TOO_LARGE,
                             'url': expose_url(url),
+                            'max_size': settings.SENTRY_SOURCE_FETCH_MAX_SIZE,
```
Comment (JTCunning) [inline src/sentry/lang/javascript/processor.py]: You could do this math here. https://github.com/getsentry/sentry/pull/4359/files#r83334946

**Review by JTCunning — APPROVED** [review]
💅

**Pull request merged**